internal/pkg/cli/app_show.go

```diff
@@ -36,25 +36,32 @@ type showAppVars struct {
 type showAppOpts struct {
 	showAppVars
 
-	store            store
-	w                io.Writer
-	sel              appSelector
-	pipelineSvc      pipelineGetter
-	newVersionGetter func(string) (versionGetter, error)
+	store             store
+	w                 io.Writer
+	sel               appSelector
+	pipelineSvc       pipelineGetter
+	deployStoreClient selector.DeployStoreClient
```

Comment:
ummm why we are using the `selector` package?

Reply:
ohh I see you are using the interface defined in `selector`. I think we should use the interface defined in `cli` package instead.

Reply:
Using interface from `cli` package now